Source/Shaders/OctahedralProjectionFS.glsl

```diff
@@ -3,8 +3,12 @@ uniform samplerCube cubeMap;
 
 void main()
 {
-    vec4 rgbm = textureCube(cubeMap, v_cubeMapCoordinates);
-    float m = rgbm.a * 16.0;
-    vec3 r = rgbm.rgb * m;
-    gl_FragColor = vec4(r * r, 1.0);
+    vec4 rgba = textureCube(cubeMap, v_cubeMapCoordinates);
+    #ifdef RGBA_NORMALIZED
+        gl_FragColor = vec4(rgba.rgb, 1.0);
+    #else
+        float m = rgba.a * 16.0;
+        vec3 r = rgba.rgb * m;
+        gl_FragColor = vec4(r * r, 1.0);
+    #endif
```

Comment:
This still looks like the RGBM encoding. Does anything produced by https://github.com/KhronosGroup/glTF-IBL-Sampler go down this path?

Reply:
nothing from the Khronos sampler reaches here - more info in above comment

Reply:
Got it, thanks!